.github/workflows/directoriesFilesChangePR.yml

```diff
@@ -57,3 +57,53 @@ jobs:
         with:
           name: output_changed_based_build_list
           path: ./**/output_*/
+  triggerGrinderTests:
+    name: Build
+    runs-on: ubuntu-latest
+    outputs:
+      test_targets_str: ${{ steps.format_test_targets_str.outputs.test_targets_str }}
+    env:
+        TARGET_LIST:  ${{ needs.getBuildLists.outputs.buildLists }}
```

Comment:
By parsing the arguments `needs.getBuildLists.outputs.buildLists` we will not need this environment variable.